games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/freegish/freegish.git#f53d915381c2dd9889dbebdff0541b04c6065067"
```

Comment:
Oh...I was looking through everywhere trying to find out where I can get such a link but didn't find it so resorted to using this.